Source/References/MenuGraphr-0.2.0.ts

```diff
@@ -692,6 +692,17 @@ declare module MenuGraphr {
     }
 
     /**
+     * A list of sounds that should be played for certain menu actions.
+     */
+    export interface ISoundNames {
+        /**
+         * The sound to play, if any, when interacting with a menu (usually off the A 
+         * or B buttons being registered).
+         */
+        menuSound?: string;
```

Comment:
The name `menuSound` isn't very clear. It's likely all sounds in this could be called `menuSound`. Go with someone more descriptive, such as `menuTick` or `onInteration`.